src/app/app.component.html

```diff
@@ -1,13 +1,12 @@
 <nav>
   <mat-toolbar>
-    <span class="brand">Prylabs</span>
-
-    <a routerLink="/" class="mat-button">Home</a>
-    <a routerLink="/stats" class="mat-button">Stats</a>
-    <a routerLink="/participate" class="mat-button">Participate</a>
+    <span class="brand">💎 Prysm Eth2 Testnet</span>
 
     <span class="flex-spacer"></span>
-    <span class="small-text" *ngIf="numValidators">{{numValidators}} validators!</span>
+    <a routerLink="/" mat-button>Home<mat-icon>home</mat-icon></a>
```

Comment:
I'm not a fan of both the text and the icon. 

I think you could have the icons on mobile only or we could use a hamburger menu. It is looking very crowded